base/logging_test.go

```diff
@@ -148,3 +149,96 @@ func BenchmarkLogRotation(b *testing.B) {
+
+	level = InfoLevel
+	assert.Equal(t, logging.Level(0x5), level.cgLevel())
+	assert.Equal(t, logging.INFO, level.cgLevel())
+	assert.Equal(t, "info", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling info logging level")
+	assert.NoError(t, level.UnmarshalText([]byte(level.String())), level.String())
+	assert.Equal(t, level.String(), ToDeprecatedLogLevel(LevelInfo).String())
+	assert.Equal(t, level.String(), ToLogLevel(*(ToDeprecatedLogLevel(LevelInfo))).String())
+
+	level = WarnLevel
+	assert.Equal(t, logging.Level(0x4), level.cgLevel())
+	assert.Equal(t, logging.WARN, level.cgLevel())
+	assert.Equal(t, "warn", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling warn logging level")
+	assert.NoError(t, level.UnmarshalText([]byte(level.String())), level.String())
+	assert.Equal(t, level.String(), ToDeprecatedLogLevel(LevelWarn).String())
+	assert.Equal(t, level.String(), ToLogLevel(*(ToDeprecatedLogLevel(LevelWarn))).String())
+
+	level = ErrorLevel
+	assert.Equal(t, logging.Level(0x3), level.cgLevel())
+	assert.Equal(t, logging.ERROR, level.cgLevel())
+	assert.Equal(t, "error", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling error logging level")
+	assert.NoError(t, level.UnmarshalText([]byte(level.String())), level.String())
+	assert.Equal(t, level.String(), ToDeprecatedLogLevel(LevelError).String())
+	assert.Equal(t, level.String(), ToLogLevel(*(ToDeprecatedLogLevel(LevelError))).String())
+
+	level = PanicLevel
+	assert.Equal(t, logging.Level(0x2), level.cgLevel())
+	assert.Equal(t, logging.SEVERE, level.cgLevel())
+	assert.Equal(t, "panic", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling panic logging level")
+	assert.NoError(t, level.UnmarshalText([]byte(level.String())), level.String())
+
+	level = FatalLevel
+	assert.Equal(t, logging.Level(0x2), level.cgLevel())
+	assert.Equal(t, logging.SEVERE, level.cgLevel())
+	assert.Equal(t, "fatal", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling fatal logging level")
+	assert.NoError(t, level.UnmarshalText([]byte(level.String())), level.String())
+
+	level = Level(0x5)
+	assert.Equal(t, logging.NONE, level.cgLevel())
+	assert.Equal(t, "Level(5)", level.String())
+	bytes, err = level.MarshalText()
+	assert.Equal(t, []byte(level.String()), bytes)
+	assert.NoError(t, err, "No error while marshalling unknown logging level")
+	assert.Error(t, level.UnmarshalText([]byte(level.String())), level.String())
+}
+
+func TestLogColor(t *testing.T) {
+	consoleLogger.ColorEnabled = false
```

Comment:
Shouldn't this be true?

```suggestion
	consoleLogger.ColorEnabled = true
```

Reply:
Yeah, you're right! This has been fixed in the latest revision.